### PR TITLE
doc improvement: nrf5340audio: tables for Audio DK

### PR DESCRIPTION
**Pull request by annwoj**
added 2 tables to nrf5340_audio/README.rst
Related to Jira NCSDK15336.

Signed-off-by: Anna Wojdylo <anna.wojdylo@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/README.rst [hunk 1]
```diff
@@ -352,6 +352,211 @@ The following figure shows the back of the development kit without the case:
 
 For the description of the relevant PCB elements, see the `User interface`_ section.
 
+.. _solderbridge_overview:
+
+Solderbridge overview
+---------------------
+
```
Comment (greg-fer) [inline applications/nrf5340_audio/README.rst:359]: We need an introductory line to each table we add.
My suggestion:
```suggestion

The nRF5340 Audio DK has a range of solder bridges for enabling or disabling selected functionalities. Changes to these are not needed for normal use of the DK.

The following table is a complete overview of the solder bridges on the nRF5340 Audio DK.

```

File: applications/nrf5340_audio/README.rst [hunk 2]
```diff
@@ -352,6 +352,211 @@ The following figure shows the back of the development kit without the case:
 
 For the description of the relevant PCB elements, see the `User interface`_ section.
 
+.. _solderbridge_overview:
+
+Solderbridge overview
+---------------------
```
Comment (greg-fer) [inline applications/nrf5340_audio/README.rst:358]:
```suggestion
Solder bridge overview
----------------------
```

File: applications/nrf5340_audio/README.rst [hunk 3]
```diff
@@ -352,6 +352,211 @@ The following figure shows the back of the development kit without the case:
...
+|SB24        | Short to bypass analog switch for MCLK                                              | Open         | Top    |
++------------+-------------------------------------------------------------------------------------+--------------+--------+
+
+
+.. _testpoint_overview:
+
+Testpoint overview
+------------------
+
++-------------+----------------------------+--------------------------------------------------+-------+--------+
```
Comment (greg-fer) [inline applications/nrf5340_audio/README.rst:418]:
```suggestion
Test point overview
-------------------

The following table is a complete overview of the test points on the nRF5340 Audio DK.

+-------------+----------------------------+--------------------------------------------------+-------+--------+
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Couple of edits requested.
@FrancescoSer , @erikrobstad , please check if you agree to them.

File: applications/nrf5340_audio/README.rst [hunk 4]
```diff
@@ -352,6 +352,211 @@ The following figure shows the back of the development kit without the case:
...
+|SB13        | Cut to disconnect digital microphone DATA from the HW CODEC                         | Shorted      | Bottom |
++------------+-------------------------------------------------------------------------------------+--------------+--------+
+|SB14        | Cut to disconnect digital microphone CLOCK from the HW CODEC                        | Shorted      | Bottom |
++------------+-------------------------------------------------------------------------------------+--------------+--------+
+|SB15        | Short to connect AUX I2S MCLK to HW CODEC MCLK1                                     | Open         | Top    |
++------------+-------------------------------------------------------------------------------------+--------------+--------+
+|SB16        | Short to connect AUX I2S MCLK to HW CODEC MCLK2                                     | Open         | Top    |
++------------+-------------------------------------------------------------------------------------+--------------+--------+
+|SB17        | Short to connect P5 pin 6 to GND	                                                   | Open         | Top    |
```
Comment (greg-fer) [inline applications/nrf5340_audio/README.rst:395]: Minor misalignment.

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (annwoj) [reply applications/nrf5340_audio/README.rst:363]: updated

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**